pfioh/swift_store.py

```diff
@@ -38,7 +38,22 @@ def _createSwiftService(self, configPath):
             'os_project_name':      config['PROJECT']['osProjectName']
         }
 
-        service = swift_service.SwiftService(options)
+        auth_swift = v3.oidc.OidcPassword(
+            options['os_auth_url'],
+            identity_provider='moc',
+            protocol='openid',
+            client_id='kaizen-client',
+            client_secret='fac377a9-f2ba-41e7-bb7f-4064dd9f4468',
```

Comment:
You shouldn't put the `client_secret` in the code. Please put it as an `OpenShift secret` and you should  consume it from the app. Get new `client_secret` and delete this one.